gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceManager.java

```diff
@@ -192,6 +193,8 @@ public class GobblinServiceManager implements ApplicationLauncher, StandardMetri
   @Getter
   @VisibleForTesting
   public DagManager dagManager;
+  @Inject
+  DagManagement dagManagement;
```

Comment:
I'd deem `DagProcEngine` the most approximate replacement for `DagManager`.  don't we want one of those here?